Preprocessing/AudioNormalizer.py

```diff
@@ -0,0 +1,104 @@
+import argparse
+from pathlib import Path
+import sys
+from typing import (
+    List,
+    Optional,
+)
+
+import ffmpeg
+
+from Preprocessing.ErrorHandlingLogger import ErrorHandlingLogger
+from Preprocessing.utils import setup_logger
+
+
+class AudioNormalizer:
+    SUPPORTED_EXTENSIONS: List[str] = [".mp4", ".mkv", ".avi"]
+
+    def __init__(self, input_folder: str, output_folder: str):
+        self.input_folder = Path(input_folder)
+        self.output_folder = Path(output_folder)
+        self.logger = ErrorHandlingLogger(
+            class_name=self.__class__.__name__,
+            logger=setup_logger(self.__class__.__name__),
+        )
```

Comment:
`self.variable` to sa non-static membery, a widze ze tylko do staticow dodales type hinty